asgard/handler.cpp

```diff
@@ -45,6 +45,23 @@ using ProjectedLocations = std::unordered_map<midgard::PointLL, valhalla::baldr:
 constexpr size_t MAX_MASK_SIZE = 10000;
 using ProjectionFailedMask = std::bitset<MAX_MASK_SIZE>;
 
+static const std::unordered_map<std::string, float> TIMECOST_DIVISOR = {{"walking", thor::kTimeDistCostThresholdPedestrianDivisor},
+                                                                        {"bike", thor::kTimeDistCostThresholdBicycleDivisor},
+                                                                        {"car", thor::kTimeDistCostThresholdAutoDivisor},
+                                                                        {"taxi", thor::kTimeDistCostThresholdAutoDivisor}};
+
+// MAX_MATRIX_Distance value set by Valhalla
```

Comment:
```suggestion
// MAX_MATRIX_Distance value set by Valhalla
// in meters
```
if it is in meters ?